edenia/antelope-tools
# Fix/environment variables 1144

### Fix environment variables

### What does this PR do?

- Resolve #1141 
    - Change the API used to query LIBRE price in USD
- Resolve #1144
    - Add HAPI_EOSRATE_GET_STATS variables to configmap
    - Remove REACT_APP_USE_CPU_BENCHMARK variable

### Steps to test

1. Run the project locally with the Libre env file.
2. Go to the Rewards Distribution.
3. Check the Exchange Rate

### Screenshots

#### **Rewards Distribution**
![imagen](https://user-images.githubusercontent.com/66583677/215836459-c099a15b-66bf-4bb3-a0e0-dcccacc51dc2.png)

#### **Undiscoverable BPs**
![imagen](https://user-images.githubusercontent.com/66583677/215836898-50478a24-a73e-4d9c-800b-9818aa4a21c1.png)
